worlds/pokemon_emerald/__init__.py

```diff
@@ -600,6 +600,10 @@ def generate_output(self, output_directory: str) -> None:
         del self.modified_starters
         del self.modified_species
 
+        # Write Output
+        out_file_name = self.multiworld.get_out_file_name_base(self.player)
+        patch.write(os.path.join(output_directory, f"{out_file_name}{patch.patch_file_ending}"))
```

Comment:
If you're changing the only os.path operation in this file anyway, can I maybe convince you that it would be in scope to change the `import os` import to `import os.path` (or if it were up to me, `from pathlib import Path`)? :D

Reply:
(In general the import of entire big python-core modules for one thing did hurt me a little bit when looking at it)

Reply:
Nvm, I don't know what I'm talking about